journal/original/16-chapter-13.adoc

```diff
@@ -941,6 +962,9 @@ Afterwards I gave forth a paper concerning True Religion, as follows:--
 [.small-break]
 '''
 
+[.embedded-content-document.paper]
```

Comment:
Can you remove the small-break above this line? (I know you did this probably before we had our most recent talk...)